nat-lab/tests/test_network_switch.py

```diff
@@ -257,7 +259,10 @@ async def test_vpn_network_switch(alpha_setup_params: SetupParameters) -> None:
                 adapter_type=telio.AdapterType.WindowsNativeWg,
                 features=TelioFeatures(direct=Direct(providers=["stun"])),
             ),
-            marks=pytest.mark.windows,
+            marks=[
+                pytest.mark.windows,
+                pytest.mark.xfail(reason="Flaky: LLT-4673"),
```

Comment:
Actually the first one should not have had the windows mark, that was a copying mistake. So I think they should be separate since they're on different OS'es.